clients/turbo.json

```diff
@@ -69,6 +69,9 @@
     },
     "openapi:generate": {
       "dependsOn": []
+    },
+    "openapi:generate-dictionary": {
```

Comment:
we'd been missing this at the turbo-level, figured it'd make sense to add?

Reply:
Yup, totally 👍